package.json

```diff
@@ -21,17 +21,17 @@
     "gatsby-plugin-robots-txt": "^1.5.0",
     "gatsby-plugin-sharp": "^2.2.31",
     "gatsby-plugin-styled-components": "^3.1.11",
-    "gatsby-plugin-tinacms": "^0.2.1",
-    "gatsby-plugin-transition-link": "^1.13.0",
+    "gatsby-plugin-tinacms": "^0.24.0",
+    "gatsby-plugin-transition-link": "^1.20.2",
```

Comment:
It seems like upgrading packages is unrelated to fixing linting errors. Why bundle them into the same pull request?